CLAUDE.md

```diff
@@ -33,6 +33,7 @@ helpful to provide project context, but only within reasonable limits.
 3. @mise.toml describes external tool dependencies
 4. @.golangci.yml contains linting configuration
 5. @.claude/commands/CLAUDE.md defines what project slash commands are available to use
+6. @docs/style-guide.md defines team coding style, and should be considered whenever making code changes
```

Comment:
The style guide I'm guessing would only apply to new code added? How do we deal with all the code that currently doesn't follow the style guide? Would we try to update it whenever we can?

Reply:
79e1f5c17286fa7f85a13558ed64fdb06b3f7cbf

I added a new section to try to clarify, LMK what you think